common.blocks/dropdown/_switcher/dropdown_switcher_button.bemhtml

```diff
@@ -12,6 +17,11 @@ block('dropdown').mod('switcher', 'button').elem('switcher').content()(function(
         resMods.size || (resMods.size = dropdownMods.size);
         resMods.theme || (resMods.theme = dropdownMods.theme);
         resMods.disabled = dropdownMods.disabled;
+
+        this._hasTick && this.isArray(res.content) && res.content.push({
```

Comment:
Let's add elem `tick` always if we get simple content. So we don't need this modifier at all.